packages/labs/cli/package.json

```diff
@@ -36,6 +36,7 @@
     "eslint": "^5.16.0",
     "globby": "^10.0.2",
     "stdout-stderr": "^0.1.13",
+    "tests": "^0.0.0",
```

Comment:
Not now, but: to make things clear, I'd like to move internal packages to under a scope, like `@lit-internal` or something.

Reply:
good idea